src/components/10-atoms/heading/index.js

```diff
@@ -30,16 +30,31 @@ class AXAHeading extends LitElement {
     // this functions applies default values per type and verifies if
     // the HTML attribute has been set before defining the custom element
     applyDefaults(this);
+
+    this.topBottomMarginsByRank = {
```

Comment:
ugh, this instance variable defines fixed constants that never change per instance! And it uses non-string keys in an object, which is illegal (and likely invisibly coerced to strings anyways).

`const TOP_BOTTOM_MARGINS_BY_RANK = {'1': '20px',.....}` outside of the class is much better.

Reply:
Yeah I tried to make this a static property but stopped at some point (didn't work at first try). You're right.